contents/blog/collaborating-on-posthog.md

```diff
@@ -0,0 +1,47 @@
+---
+date: 2022-03-24
+title: Collaborating on PostHog
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["paolodamico"]
+featuredImage: ../images/blog/generic-release-notes.png
+featuredImageType: full
+categories: ["Product updates"]
+---
+
+PostHog is better accompanied. We quickly realized that as more users join an organization, the incremental value derived by each user increases. Consider for instance when you create a dashboard. You only create it once, but then with each user that joins your organization, the dashboard becomes more valuable. We found clear signals of this in our data: organizations with multiple teammates retain on average 30%+ better. We also recognized that conservatively we could increase insight discovery (our [main Product metric](https://posthog.com/handbook/product/metrics)) by more than 20% by improving collaboration workflows.
```

Comment:
> PostHog is better accompanied

Not sure this makes gramatical sense.

> We also recognized that conservatively we could increase insight discovery (our [main Product metric](https://posthog.com/handbook/product/metrics)) by more than 20% by improving collaboration workflows.

We should maybe give a bit more detail on how we estimated this opportunity size.

Reply:
Added more context.